### DISCUSSION TITLE
NFT smart contract

### DISCUSSION THREAD

**Discussion by BEBE1M** [smartcontractkit/full-blockchain-solidity-course-py#1215] [Q&A]
Hi guys,i have few questions about smart contract. Is it possible to make mint web,where each NFT will have their own price and mint separately

**Comment by cromewar** [comment]
Hello @BEBE1M 
Well the concept of mint is to generate a random NFT from the pool so you are not going to know which one you are getting, if you want to set a different price for each you are looking for a NFT store like opensea.

**Reply by BEBE1M** [reply]
first of  #all, thank you for replying. i know, the main Idea of smart contract with many features is to have random mint which is the main intrigue of the project but what that i working is different. we don't need this type of mint page. people should choose which one they prefer and each of them must have their own price.

**Reply by cromewar** [reply]
Well is complicated as you would have to setup a unit price for each individual NFT, a good approach is to do something like this, let's suppose you have a total of 90 NFTS that you want to sell.
1. You can separate them based on their ID or or breed, or rareness, let's say 3 groups of 30.
2. you can separate each group on a range of prices, the most common ones should be cheaper and the rarest one expensive.
3. you allow the user to choose on wich tier the NFT should be.

Anyways remember that the _mint function takes the ID as parameter, so you can just use the ID to mint each one, but as I said if you use a big collection  like 5000 NFTS, this could be harder.

**Reply by JohnEMagee** [reply]
Why would you want a project like this, what's the different process?
